build.gradle

```diff
@@ -35,7 +36,7 @@ dependencies {
 
     compileOnly("org.projectlombok:lombok:${lombokVersion}")
     testCompile("org.springframework.boot:spring-boot-starter-test:${springBootVersion}")
-    testCompile("org.spockframework:spock-spring:${spockSpringVersion}")
+    testCompile("org.spockframework:spock-spring:1.2-groovy-2.5")

```

Comment:
Dependency updates should be done by increasing the version of fint-buildscripts to https://raw.githubusercontent.com/FINTlibs/fint-buildscripts/v1.8.0/dependencies.gradle

Reply:
Agree